### PR TITLE
MSolve Initial Commit for koralli integration

### PR DESCRIPTION
**Pull request by TheoChristo**

### CODE REVIEW

**Comment by finkandreas** [comment]
Looks fine to me, I have tried to reference it in my [PR4](https://github.com/DComEX/dcomex-prototype/pull/4). You can review `Dockerfile` there to see if the last commented section seems correct to build the right MSolve module.
Dotnet is installed in the base container `finkandreas/dcomex-prototype_base:1.0`. See my comment that this is not the final solution, there will be a pipeline to update the base image when necessary.

**Review by finkandreas — APPROVED** [review]

File: msolve/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:18.04
```
Comment (bcumming) [inline msolve/Dockerfile]: Now that the dockerfiles that define the image we will build-test-deploy on Daint are in the `ci` path, this isn't needed.

File: msolve/runTest.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (bcumming) [inline msolve/runTest.sh:1]: On these tests:
- should they test the integration of Korali+MSolve
- should they be in a different path (e.g. `ci` or future `integration` path?)

**Review by bcumming — CHANGES_REQUESTED** [review]
A couple of comments regarding the location of build recipes for ci and the tests now that the focus is on integration.

**Pull request merged**